### PR TITLE
more test coverage

### PR DESCRIPTION
**Pull request by blinkinglight**

### CODE REVIEW

File: tokens.go [hunk 1]
```diff
@@ -114,13 +114,13 @@ func (t *Tokens) tokens() {
 func (t *Tokens) remoteGet(url string) []byte {
 	response, err := http.Get(url)
 	if err != nil {
-		log.Printf("remote_get error: %v", err)
+		// log.Printf("remote_get error: %v", err)
```
Comment (nycterent) [inline tokens.go]: Kas če?

File: tokens.go [hunk 2]
```diff
@@ -114,13 +114,13 @@ func (t *Tokens) tokens() {
...
 		return nil
 	}
 	defer response.Body.Close()
 	b, err := ioutil.ReadAll(response.Body)
 	if err != nil {
-		log.Printf("remote_get error: %v", err)
+		// log.Printf("remote_get error: %v", err)
```
Comment (nycterent) [inline tokens.go]: Ir če?

File: tokens.go (hunk 1 above)
Comment (blinkinglight) [reply tokens.go]: ups ;)

File: tokens.go (hunk 2 above)
Comment (blinkinglight) [reply tokens.go]: also ups; )

**Review by nycterent — APPROVED** [review]

**Pull request merged**